### ISSUE TITLE
[KEYBOARD]: Outreach materials - focus should be set on the number of results on pagination keypress or click

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#981]
## Description

Keyboard focus is being lost to the `<body>` tag when users click on any of the pagination links at the bottom of the screen. We should set focus back to the number of results so keyboard and screen reader users do not have to navigate back to the top of the page manually. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As a keyboard user, I want to focus the first result card link by pressing `TAB` once after I have clicked a pagination link
* As a screen reader user, I want the same as above, and want to hear the search results read aloud

## Environment
* MacOS Mojave
* Safari
* VoiceOver

## Possible Fixes (optional)
By adding a `tabindex` attribute, we can set focus programmatically. I think the VA helper method for setting focus could be added to the pagination click handler(s).

```html
<!-- Add a tabindex="-1" to set focus with JavaScript -->
<div class="break-div" tabindex="-1">
  Showing
  <span id="total-pages">10</span>
  <span id="total-all"> of 176</span> results
</div>
```

## WCAG or Vendor Guidance (optional)
* [Keyboard: Understanding SC 2.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html)

## Screenshots or Trace Logs

![Screen Shot 2019-08-14 at 11.44.50 AM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/729e3178-953a-4ea5-a220-b1bee0c60da8)

**Comment by jefflbrauer** [comment]
@1Copenut Are you now complete with the 508 review? I thought as of Tuesday we were all set but then yesterday saw 2 new tickets come in including this one. Just want to batch all remaining issues for my team to address. 

Thanks,
Jeff

**Comment by 1Copenut** [comment]
@jefflbrauer I have one small one (lower priority) I'm entering now, and that's the last one. I've got an open question I'm taking to the 508 office meeting today, but it's not an issue, just an improvement, so will not be issuing a ticket.

Last item will be posted in just a moment, then I'll close the audit.

**Comment by jefflbrauer** [comment]
@1Copenut please review again

cc @ethanteague

**Comment by 1Copenut** [comment]
@jefflbrauer The Next, Previous, First and Last links are working mostly as expected. When I click them or press `ENTER` using the keyboard, focus is moved up to the number of results. 

There are a few items that we should address before closing this ticket:

1. Clicking the page numbers changes the URL in the address bar, but does not load the next / previous batch of results. This is happening with screen readers, with the keyboard, and with the mouse click.
2. The Next, Previous, First, Last links are moving focus and updating the content, but they are not updating the URL bar.
3. JAWS and VoiceOver are not honoring the new focus spot on the number of results when I am navigating using arrow keys in JAWS (virtual cursor) or using the `Ctrl + Opt + arrow` chord in VoiceOver (cursor). I think this has to do with the structure of the HTML, but will need to dig into it further tomorrow and advise.

**Comment by 1Copenut** [comment]
@jefflbrauer Is this item still being worked on, or should I make time to re-test today / Monday?
